### DISCUSSION TITLE
Giving more details during PIN and password creation

### DISCUSSION THREAD

**Discussion by IcecolD120** [iotaledger/firefly#1574] [Features / Enhancements]
Hi Team,

That would be great if we can have more information when creating the PIN Code and the password.
For example:
- The password will be used to lock/unlock stronghold + what is stronghold.
- The pin code will be used to open the wallet 

Thank you,
Ice colD

**Comment by rafaeldjpbrochado** [comment]
I would suggest not mentioning Stronghold at all. Keep it simple.

Perhaps something along the lines of "The password will be used to unlock your wallets, which then allows you to send money. You can still check your balance when the wallets are locked, and it is good practice to keep them locked when you're not sending funds".

As for the PIN: "The PIN code will be used to log onto your profile, where your wallets are kept".

It might not be the most accurate way to describe the underlying tech, but this is how I would explain it to users who are unfamiliar with the tech.

Edit: Also, it might be the case that this is already in Firefly? It's been a while since I've created a new profile. :P
